twake/frontend/src/app/features/channel-members/hooks/use-channel-guests.ts

```diff
@@ -31,6 +34,11 @@ export const useChannelGuests = (
     );
 
     if (channelGuestsUpdated) _setChannelGuests(channelGuestsUpdated);
+
+    const usersIdx = channelGuestsUpdated.map(guest => guest.user_id || '');
+    const users = await UserAPIClient.list(usersIdx, [companyId]);
```

Comment:
Not sure about doing another call to backend here could we put a Fixme comment ?

Reply:
done